### PR TITLE
Add crash stats section to ClusterFuzz's documentation.

### PR DESCRIPTION
**Pull request by tanin47**
The crashes shown in the image are public.

### CODE REVIEW

File: docs/clusterfuzz.md [hunk 1]
```diff
@@ -43,4 +43,8 @@ all cited issues, so as to keep your fuzz target running efficiently and finding
 
 ![performance_analyzer](https://raw.githubusercontent.com/google/oss-fuzz/master/docs/images/expat_performance_analyzer.png)
 
+## Crash stats
 
+You can view statistics of crashes over time on our crash statistics dashboard.
+
+![crash_stats](https://raw.githubusercontent.com/google/oss-fuzz/master/docs/images/crash_stats.png)
```
Comment (oliverchang) [inline docs/clusterfuzz.md]: nit: could you do 
```
![crash_stats](docs/images/crash_stats.png?raw=true)
```
instead?

Comment (tanin47) [inline docs/clusterfuzz.md]: Yeah, that's better. I changed every other image's url as well.

Comment (oliverchang) [inline docs/clusterfuzz.md]: Thanks!

**Comment by inferno-chromium** [comment]
We need a good example from oss-fuzz repo, not chromium.
e.g.
https://oss-fuzz.com/v2/crash-stats?end=414743&hours=1&job=libfuzzer_asan_wireshark&order=total_count&security=no

**Pull request merged**